shenanigan/models/stackgan/run.py

```diff
@@ -105,6 +106,21 @@ def run(train_loader, val_loader, small_image_dims, results_dir, settings, stage
         plotter = LogPlotter(results_dir)
         plotter.learning_curve()
 
+    if stage == 1 and evaluate:
+        raise NotImplementedError('Evaluation for stage 1 is not implemented')
```

Comment:
Maybe do something like this for stage 2 as well if we can't evaluate stage 2

Reply:
one of the previous checks will catch stage 2 and retained so figured that was fine